c7n/resources/appelb.py

```diff
@@ -1372,3 +1372,143 @@ def process_target_group(self, client, target_group):
         self.manager.retry(
             client.delete_target_group,
             TargetGroupArn=target_group['TargetGroupArn'])
+
+
+class TargetGroupAttributeFilterBase:
+    """ Mixin base class for filters that query Target Group attributes.
+    """
+
+    def initialize(self, tgs):
+        client = local_session(self.manager.session_factory).client('elbv2')
+
+        def _process_attributes(tg):
+            if 'Attributes' not in tg:
```

Comment:
this should be an annotation ala `c7n:TargetGroupAttributes`

Reply:
@thisisshi annotation has been changed from 'Attributes' to 'c7n:TargetGroupAttributes'